composer.json

```diff
@@ -46,25 +62,13 @@
 			"BracketSpace\\Notification\\Tests\\": "tests"
 		}
 	},
-	"require": {
-		"micropackage/requirements": "^1.0",
-		"micropackage/dochooks": "^1.0",
-		"micropackage/filesystem": "^1.1",
-		"micropackage/templates": "^1.1",
-		"micropackage/ajax": "^1.0",
-		"typisttech/imposter-plugin": "^0.6.2"
-	},
-	"extra": {
-		"imposter": {
-			"namespace": "BracketSpace\\Notification\\Vendor",
-			"excludes": [
-				"roots/wordpress",
-				"wp-phpunit/wp-phpunit"
-			]
-		},
-		"wordpress-install-dir": "tests/_wordpress"
-	},
-	"config": {
-        "platform-check": false
-    }
+	"scripts": {
+		"phpcbf": "phpcbf",
+		"phpcs": "phpcs",
+		"phpcompat": "phpcs --standard=PHPCompatibility --runtime-set testVersion 7.0 notification.php load.php uninstall.php src/",
```

Comment:
Also this line.

Reply:
@szepeviktor couldn't this run as a part of phpcs? We even have this thing [configured inside phpcs.xml](https://github.com/BracketSpace/Notification/blob/develop/phpcs.xml#L34)

Reply:
BTW It passes: we are 7.0 compatible!

Reply:
> We even have this thing [configured inside phpcs.xml](https://github.com/BracketSpace/Notification/blob/develop/phpcs.xml?rgh-link-date=2021-03-15T19%3A32%3A54Z#L34)

Okay! The Composer script is for running it separately.
Are our computers big enough to carry this 1 line? :)

Reply:
Hopefully :)